### ISSUE TITLE
can not reproduce the detection result in paper for WIDER hard subset

### ISSUE THREAD

**Issue by kiloeye** [mahyarnajibi/SSH#11]
@mahyarnajibi,
I tried detection using the latest code with model in download_ssh_model.sh. On WIDER validation set I got 92.5/90.9/73.7 respectively for easy/medium/hard subset. That is not consistent with results in paper: 91.9/90.9/81.4. Particularly for hard subset, the gap(81.4 vs. 73.7) is significant.
I also trained the model from scratch, the detection performances are 92.3/90.5/72.7. Result on hard subset differ much from 81.4.
Is there any specific setting for hard subset I am missing?
thank!

**Comment by mahyarnajibi** [comment]
Hi,
So you just cloned the repo and ran main_test.py with the provided model? Did you change any parameters in SSH/configs/default_config.yml or SSH/configs/wider.yml? Also the dataset is directly downloaded from http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/ ?
The provided model with the default config should produce the same AP as the paper.

**Comment by kiloeye** [comment]
@mahyarnajibi,
I found out the gap is due to inpuy image resolution. when I resize image to 1200x1600, the results are the same as in the paper .
thank you! it is a great job!

**Comment by xinyaoguo** [comment]
@kiloeye How do you resize an image?

**Comment by AruniRC** [comment]
@kiloeye you mean the test-time resolution is 1200x1600 ? But this would change the aspect ratio of the images... which could actually lower the results.

**Comment by loackerc** [comment]
@xinyaoguo, you can check the config file where you define the scales and max_size.
@AruniRC, the aspect ratio does not change. If you look at the code, the interpolation is made such that the aspect ratio is kept, while limiting the long edge to 1600.
